perftracker/templates/comparison_id.html

```diff
@@ -242,8 +242,12 @@ <h4>Job: {{ job }}</h4>
         serie: sect_data[s_id].series[0].data,
         less_better: sect_data[s_id].series[0].less_better,
     };
-    $('input:checked').each(function () {
-        labeled_data[s_id][$(this)[0].name] = $(this)[0].value;
+
+    $('input:checked').filter(function () {
+        return $(this)[0].id.includes(s_id.toString());
```

Comment:
error-prone filtering, s_ids on the same comparison may look like     123 and 10123